README.md

```diff
@@ -8,6 +8,41 @@
 > -- _ChatGPT, personal communication, February 2024._
 
 ## Overview
+Diachronic provides a way to make Temporal nondeterminism errors a thing of the past.
+
+### How it works
+Workflows take their current state and pass it to the next version of the program. The new program receives it and continues where the old program left off, subject to a user-defined transformation.
+
+Say, for some reason, you have a program that models a toaster. The toaster can be in one of two states, “on” or “off” — on when the timer is running and it’s plugged in, off otherwise. It also has a counter for number of toasts.
+
+We can model this with the following state transition diagram:
+off -> plugged-in -> on
+off -> timer on -> on
+on -> timer off? -> off
+on -> unplug-it -> off
+
+You deploy to prod. Toaster run around the world. And there was much rejoicing.
+
+Naturally, there’s a bug. Simply because the toaster is plugged in doesn’t mean there’s power. You now need to change a durable program that models the live state of thousands of devices.
```

Comment:
```suggestion
But there’s a bug. Simply because the toaster is plugged in doesn’t mean there’s power. This means need to change a workflow that has been modeling the live state of thousands of devices in a way that could break their history. 
```